pkg/templates/danglingingress/template.go

```diff
@@ -0,0 +1,112 @@
+package danglingingress
+
+import (
+	"fmt"
+
+	"golang.stackrox.io/kube-linter/pkg/check"
+	"golang.stackrox.io/kube-linter/pkg/config"
+	"golang.stackrox.io/kube-linter/pkg/diagnostic"
+	"golang.stackrox.io/kube-linter/pkg/lintcontext"
+	"golang.stackrox.io/kube-linter/pkg/objectkinds"
+	"golang.stackrox.io/kube-linter/pkg/templates"
+	"golang.stackrox.io/kube-linter/pkg/templates/danglingingress/internal/params"
+	v1 "k8s.io/api/core/v1"
+	networkingV1 "k8s.io/api/networking/v1"
+)
+
+const (
+	templateKey = "dangling-ingress"
+)
+
+func getSelectorsFromIngressBackend(b *networkingV1.IngressBackend) string {
+	service := b.Service
+	if service == nil {
+		return ""
+	}
+
+	return service.Name
+}
+
+func getSelectorsFromIngress(ingress *networkingV1.Ingress) map[string]bool {
+	selectors := map[string]bool{}
```

Comment:
I'd favor `map[string]struct{}` and setting `selectors[s] = struct{}{}` subsequently:
```suggestion
	selectors := map[string]struct{}
```

Reply:
:+1: